floyd/cli/experiment.py

```diff
@@ -182,12 +182,33 @@ def get_log_id(job_id):
     return instance_log_id
 
 
+def check_job_termination_line(log_output_line):
+    """
+    Get Job termination by inspecting if any of the termination log lines is detected
+    in the current log output line.
+    """
+    job_terminated = False
+    success = "[success] Finished execution"
+    failure = "[failed] Task execution failed"
+    shutdown = "[shutdown] Task execution cancelled"
+    timeout = "[timeout] Task execution cancelled"
+    termination_list = [success, failure, shutdown, timeout]
+
+    if any(terminal_line in log_output_line for terminal_line in termination_list):
```

Comment:
for simplification, you can just write `return any(terminal_line in log_output_line for terminal_line in termination_list)`.